stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -143,6 +143,8 @@ public void prepare() throws Exception {
       hdfsConf.set(entry.getKey(), entry.getValue());
     }
 
+    hdfsConf.set(PropertyKey.Name.USER_FILE_WRITE_TYPE_DEFAULT, mParameters.mWriteType);
```

Comment:
Here the write type is set explicitly to make sure the operation is done with the desired type.